rmw_dds_common/include/rmw_dds_common/graph_cache.hpp

```diff
@@ -498,10 +499,18 @@ class GraphCache
    * @}
    */
 
+  /// Internal, do not use.
```

Comment:
@ahcorde @ivanpauno FYI, there is an [\internal](https://www.doxygen.nl/manual/commands.html#cmdinternal) Doxygen command.

Reply:
Done https://github.com/ros2/rmw_dds_common/pull/26/commits/462c77547848704b78218103ae068db6a666e728